tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -77,3 +115,40 @@ def test_wazuh_db_create_agent(configure_sockets_environment, connect_to_sockets
                            "stage": "Syscheck - Agent does not exits yet"}]}
     test_wazuh_db_messages(configure_sockets_environment, connect_to_sockets_module, test['test_case'])
     assert os.path.exists(os.path.join(WAZUH_PATH, 'queue', 'db', "999.db"))
+
+
+def test_wazuh_db_chunks(configure_sockets_environment, connect_to_sockets_module):
+    """Check that commands by chunks work properly when agents amount exceed the response maximum size"""  
+    AGENTS_CANT = 14000
+    AGENTS_OFFSET = 20
+    AGENTS_MAX = AGENTS_OFFSET+AGENTS_CANT
+    
+    #Pre insert agents
+    insert_agents(AGENTS_OFFSET, AGENTS_CANT)
+   
+    #Check get-all-agents chunk limit
+    command = f'global get-all-agents last_id 0'
+    receiver_sockets[0].send(command, size=True)
+    response = receiver_sockets[0].receive(size=True).decode()
+    
+    status = response.split(" ",1)[0]
+    assert status == 'due', 'Failed chunks check on get-all-agents. Expected: {}. Response: {}'\
+           .format('due', status)
+    
+    #Check get-agents-by-keepalive chunk limit
+    command = f'global get-agents-by-keepalive condition > -1 last_id 0'
+    receiver_sockets[0].send(command, size=True)
+    response = receiver_sockets[0].receive(size=True).decode()
+    
+    status = response.split(" ",1)[0]
+    assert status == 'due', 'Failed chunks check on get-agents-by-keepalive. Expected: {}. Response: {}'\
+           .format('due', status)
+   
+    #Check sync-agent-info-get chunk limit
+    command = f'global sync-agent-info-get last_id 0'
+    receiver_sockets[0].send(command, size=True)
+    response = receiver_sockets[0].receive(size=True).decode()
+    
+    status = response.split(" ",1)[0]
+    assert status == 'due', 'Failed chunks check on sync-agent-info-get. Expected: {}. Response: {}'\
+           .format('due', status)
```

Comment:
Refactor this to use Python inner functions.

Reply:
Done